src/instructlab/model/chat.py

```diff
@@ -586,6 +587,12 @@ def chat_model(
     visible_overflow,
 ):
     """Runs a chat using the modified model"""
+    if not FeatureGating.feature_available(GatedFeatures.RAG):
+        raise ValueError(
+            f"In order to use experimental features, set {FeatureGating.env_var_name}"
+            f" to one of: {FeatureGating.available_scopes()}"
+        )
+
```

Comment:
Same comment about the error message as above - this could be more specific and helpful to the user, such as telling them which exact feature scope to use and let them which experimental feature they tried to use that we're blocking unless they enable the gate.

Reply:
That's a good point. There is a risk of those error messages going out of date by being forgotten to be changed - but that can be addressed with unit tests, which I'll add now.

Reply:
Done.